### ISSUE TITLE
[Question] about the reported entity number on wikidata

### ISSUE THREAD

**Issue by jayzzhou-thu** [facebookresearch/PyTorch-BigGraph#36]
Hello, thanks for your excellent work on large-scale pretrained graph embeddings. But I have a small question here:

The reported number of entities on Wikidata is 78 million. However, the number on the Wikidata website is only 56 million and I only extracted 54 million entities using the April 2019 dump. Can you give some details about how you extract entities from the dump? 

Thanks!

**Comment by lerks** [comment]
@ledw should be able to give you a definite answer, but maybe it's due to what she mentioned in https://github.com/facebookresearch/PyTorch-BigGraph/issues/15#issuecomment-479951925?

**Comment by ledw** [comment]
@jayzzhou-thu as mentioned by @lerks above, in our model we have both entities and labels from 
Wikidata. In addition, we filtered out entities and labels that appeared less than 5 times in the data dump.
